bugtool/cmd/root.go

```diff
@@ -265,6 +267,11 @@ func cleanup(dbgDir string) {
 			files = append(files, fmt.Sprintf("%s.tar", dbgDir))
 		case "tar":
 			files = append(files, dbgDir)
+
+		case "o":
+			if excludeObjectFiles {
+				files = append(files, dbgDir)
+			}
```

Comment:
I don't think this is the correct place to exclude these files. `cleanup` is used to remove temporary files created during creation of the sysdump.

Reply:
@aanm Could you please inform me whether my understanding of why the current implementation is wrong is reasonable?

And could you please give some idea on where the code should be to exclude the object files from `sysdump` by default.

Reply:
@pchaigno can you take a look? Thanks

Reply:
@NikhilSharmaWe Did you test this? Tobias is correct, this isn't the right place to remove those files and I'd be very surprised if it worked.

The state directory contents, including object files, are copied at https://github.com/cilium/cilium/blob/e0c07f3d10dc47c09d95aecdb4eb28307ecabc94/bugtool/cmd/configuration.go#L408 and https://github.com/cilium/cilium/blob/e0c07f3d10dc47c09d95aecdb4eb28307ecabc94/bugtool/cmd/configuration.go#L418-L421
We may be able to simply add a command to the list, after the `kubectl cp`, to remove the `.o` files in the copied directory.